lib/rdf2marc/rdf2model/mappers/leader.rb

```diff
@@ -17,6 +17,18 @@ def generate
 
         private
 
+        def record_status
+          item.admin_metadata.query.path_first_literal([[BF.status, BF.Status], BF.code])
+        rescue Rdf2marc::MappingError => e
+          return if e.message.match?(status_not_literal_pattern)
```

Comment:
Eat the error and return nil and allow the `Leader` struct to handle defaulting the record status.